tests/lib/mongo/repositories/uploads.test.ts

```diff
@@ -0,0 +1,50 @@
+import { config } from 'dotenv';
+import { MongoDBUploadsRepository } from '../../../../lib/core/uploads/MongoDBUploadsRepository';
+import { uploadFixtures } from '../fixtures/uploads.fixtures';
+import { unloadLoadFixtures } from '../fixtures/init-fixtures';
+import { setupAndValidateStorageForFixtures } from './utils';
+
+config();
+
+const { storage, uri, BRIDGE_TEST_DB_NAME } =
+  setupAndValidateStorageForFixtures();
+
+let repository: MongoDBUploadsRepository = new MongoDBUploadsRepository(
+  storage.models.Upload
+);
+
+const [upload1, upload2] = uploadFixtures;
+
+beforeEach((ready) => {
+  unloadLoadFixtures(uri, BRIDGE_TEST_DB_NAME).then(() => {
+    ready();
+  });
+});
+
+afterAll((finish) => {
+  unloadLoadFixtures(uri, BRIDGE_TEST_DB_NAME).then(() => {
+    storage.connection.close();
+    finish();
+  });
+});
+
+describe('Uploads repository', () => {
+  it('findByUuids()', async () => {
```

Comment:
Try to group unit tests of the same method by `describe`:

```
describe('findByUuids()', () => {
  it('Should ....', () => {...})
  it('Should ....', () => {...})
})

it('findById', () => {
  // just one test, could be an 'it'
  // if more than one tests then group by describe better
})
```

Reply:
fixed